src/index.js

```diff
@@ -21,6 +22,7 @@ const App = () => {
       // Cart state
       products: [],
       totalPrice: 0,
+      customerId: null,
```

Comment:
Please move this out of the `cart` state in the `easy-peasy` store into a new state called `customer` in the store. It does not belong there... as the cart is for the cart products and price...

Reply:
fixed